kubemarine/haproxy.py

```diff
@@ -72,7 +72,7 @@ def install(group):
     with RemoteExecutor(group.cluster) as exe:
         for node in group.get_ordered_members_list(provide_node_configs=True):
             package_associations = group.cluster.get_associations_for_node(node['connect_to'])['haproxy']
-            group.sudo("%s -v" % package_associations['executable_name'], warn=True)
+            group.sudo("bash %s -v" % package_associations['executable_name'], warn=True)
```

Comment:
We need to find root cause. I do not know what happens on ubuntu 22, but on 20 it stops working
```
ubuntu@k8s-balancer-left:~$ /usr/sbin/haproxy -v
HA-Proxy version 2.0.29-0ubuntu1 2022/08/26 - https://haproxy.org/
ubuntu@k8s-balancer-left:~$ bash /usr/sbin/haproxy -v
/usr/sbin/haproxy: /usr/sbin/haproxy: cannot execute binary file 
```